Tools/MiniBrowser/playstation/MainWindow.h

```diff
@@ -50,6 +51,7 @@ class MainWindow final : public toolkitten::Widget {
     void setActiveWebView(WebViewWindow*);
     void switchActiveWebView(int delta);
     void closeWebView(WebViewWindow*);
+    void parseOptions(const std::vector<std::string>& options, std::string* requestedURL);
```

Comment:
Could we just have it return a `std::string` instead of doing the pointer to `std::string`?

Reply:
How about an idea of initializing `requestedURL` as "https://webkit.org", and pass the reference of the string to parseOptions?